### ISSUE TITLE
Chest opening question

### ISSUE THREAD

**Issue by thatguy692** [kolton/d2bot-with-kolbot#1626]
Ok i'd like to open sparkly chest in mausoleum
i don't know how to proceed

**Comment by Spoofzor** [comment]
There are a couple ways to do this.

In character config, if you have mausoleum = true
near Line 432, `Config.OpenChests = true`
You can also do `Config.OpenChests = 2`

This will open chests during the script. Setting 2 opens all containers (tombs, dead rogues, chests, etc)

Alternatively, if you do not clear mausoleum you can use ChestMania near line 198
I do not have the area code for mausoleum, but it shouldnt be too hard to find. It's in one of the text files

**Comment by DarkHorseDre** [comment]
areas stored in text file: https://github.com/kolton/d2bot-with-kolbot/blob/master/d2bs/kolbot/sdk/areas.txt

note that setting openchests to 2 will use the wider (but not fully complete) container list, which will increase run times and can be dangerous in some areas if you dont have a tough toon (e.g. late act 3)

Watch the bot runs after changing this to ensure it doesnt have adverse effects

**Comment by Spoofzor** [comment]
@DarkHorseDre Very good point. I forgot to mention two things: 

1. I ended up removing barrels from the list because the bot would spend 10 minutes of every run smashing all the barrels in act 1. Pretty funny to watch the first time, actually.

2. Pretty sure I saw this fix in a previous issue and it was DarkHorseDre that pointed out the solution in my original comment.  He is the miracle worker

**Comment by DarkHorseDre** [comment]
yeah lol funny, but also theres a delay on kolbot as it tries to click so can be a pain.

remove jars and urns too ^^

haha not at all mate - I just get lucky sometimes ;)

**Comment by loldermond** [comment]
So how do i remove b arrels jars and urns?
